PowerEditor/src/WinControls/ProjectPanel/ProjectPanel.cpp

```diff
@@ -306,6 +376,42 @@ BOOL ProjectPanel::setImageList(int root_clean_id, int root_dirty_id, int projec
 	ImageList_AddMasked(_hImaLst, hbmp, maskColour);
 	DeleteObject(hbmp);
 
+	hbmp = LoadBitmap(_hInst, MAKEINTRESOURCE(open_dir_id));
+	if (hbmp == NULL)
+		return FALSE;
+	ImageList_AddMasked(_hImaLst, hbmp, maskColour);
+	DeleteObject(hbmp);

```

Comment:
Both `ImageList_AddMasked` and `DeleteObject` can fail. It's probably a good idea to check the return value, and `throw std::logic_error` on failure?